osmaxx/api_client/conversion_api_client.py

```diff
@@ -129,6 +129,18 @@ def _create_job_TODO_replace_me(self, extraction_order, request):  # noqa
             logging.error('Could not retrieve api job id from response.', response)
         return response
 
+    def get_result_file_url(self, job_id):
+        job_detail_url = CONVERSION_JOB_URL + '{}/'.format(job_id)
+        return self.authorized_get(job_detail_url).json()['resulting_file']
+
+    def download_file_to(self, url, destination_filename):
+        r = self.authorized_get(url, stream=True)
+        with open(destination_filename, 'wb') as f:
+            for chunk in r.iter_content(chunk_size=1024):
+                if chunk:
+                    f.write(chunk)
```

Comment:
And is there any benefit to this manual chunking?

And why is the file written to disk here and read from disk and written again [in the caller](https://github.com/geometalab/osmaxx-frontend/pull/459/files#diff-e52c21d719a322835dc4db89ea595076R67)? Wouldn't it make more sense to pass it in-memory (e.g. as `ContentFile`)?